config/codon_xlarge.config

```diff
@@ -1,108 +1,146 @@
 process {
+
+    // TODO: The time needs to be adjusted
+    time = '5d'
+
     withName: ANNONTATE_GFF {
-        cpus = 1; memory = { 1.GB * task.attempt }
+        cpus = 1;
```

Comment:
not sure if it matters - is the semi-colon here a problem? it's removed everywhere else

Reply:
It does! Good catch